integration-tests/modules/__tests__/cart/store/carts.spec.ts

```diff
@@ -2015,16 +2053,16 @@ medusaIntegrationTestRunner({
             type: "order",
             order: expect.objectContaining({
               id: expect.any(String),
-              total: 106,
+              total: 94.764,
```

Comment:
@carlos-r-l-rodrigues can you help check the totals for a cart after applying a promo? does this look correct?

Reply:
I'm not sure, it seems that this is a fixed amount, but it is considering a percentage?

Reply:
yeah exactly, this seems to be coming from the tax totals that are all over the place. I'll tackle this when I tackle the tax ticket for promotions.